utils/SwiftIntTypes.py

```diff
@@ -122,7 +122,7 @@ def all_integer_or_real_binary_operator_names():
 
 
 def all_integer_assignment_operator_names():
-    return ['%=', '<<=', '>>=', '&=', '^=', '|=']
+    return ['%=', '<<=', '>>=', '&*=', '&=', '&+=', '&-=', '^=', '|=']
```

Comment:
I don't think these changes are required here. For example, `&<<=` isn't present.